app/index.js

```diff
@@ -91,13 +94,13 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'email',
         message: 'Your email (optional):',
         default: this.options.email || this.gitc.user.email,
-        when: !this.options.email
+        when: this.options.email === null || this.options.email === undefined
```

Comment:
Let's just do `this.options.email == null` to make it less verbose

Reply:
For name, email and website? Or which one exactly?

Reply:
For all cases

Reply:
eslint didn't like a == null.

a === null actually doesn't happen. So I used

```
a === undefined
```

And the tests are now ok.